homeassistant/components/emoncms/config_flow.py

```diff
@@ -48,13 +48,19 @@ def sensor_name(url: str) -> str:
     return f"emoncms@{sensorip}"
 
 
-async def get_feed_list(hass: HomeAssistant, url: str, api_key: str) -> dict[str, Any]:
+async def get_feed_list(
+    hass: HomeAssistant,
+    url: str,
+    api_key: str,
+    emoncms_client: EmoncmsClient | None = None,
```

Comment:
I think it's better to make it compulsory instead.
```suggestion
    emoncms_client: EmoncmsClient,
```

Reply:
my bad, you are right
done